plugin/source/quick_select.cpp

```diff
@@ -40,6 +42,24 @@ static void cycleQuickSelect()
     }
 }
 
+
+static void toggleEmulation()
+{
+    NfpiiEmulationState state = NfpiiGetEmulationState();
+    std::string notifText;
+    if (state == NFPII_EMULATION_ON) {
+        NfpiiSetEmulationState(NFPII_EMULATION_OFF);
+        std::string notifText = "re_nfpii: Disabled emulation";
```

Comment:
You need to remove the `std::string` here, otherwise you're creating a new string in the current scope.
```suggestion
        notifText = "re_nfpii: Disabled emulation";
```